aten/src/ATen/native/mps/operations/MultiTensorApply.h

```diff
@@ -153,6 +157,54 @@ struct FusedSgdEncodingFunctor {
   }
 };
 
+template <>
+struct FusedSgdEncodingFunctor<false> {
+  void operator()(
+    id<MTLComputeCommandEncoder>& computeEncoder,
+      id<MTLBuffer>& tensorArgumentBuffer,
+      const MetadataArguments& metadata_arguments,
+      const double weight_decay,
+      const double lr,
+      const bool maximize
+    ) const {
+      float weight_decay_lv = weight_decay;
+      float lr_lv = lr;
+      uint8_t maximize_lv = maximize;
+
+      [computeEncoder setBuffer:tensorArgumentBuffer
+                                  offset:0
+                                  atIndex:0];
+      [computeEncoder setBytes:&metadata_arguments
+                                  length:sizeof(MetadataArguments)
+                                  atIndex:1];
+      [computeEncoder setBytes:&weight_decay_lv length:sizeof(float) atIndex:2];
```

Comment:
Consider using convenience methods
```suggestion
      mtl_setBytes(computeEncoder, weight_decay_lv, 2);
```

Reply:
It seems that we only have `mtl_setBuffer` and don't have `mtl_setBytes` method. :P

Reply:
Oh just saw you committed last week.

Reply:
I feel like keeping the current state for this file is better for code reading consistency since `mtl_setBytes`'s templates do not support custom sturct types if I understand correctly?

Reply:
done